server/StudySharp.API/appsettings.Development.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
```

Comment:
Revert these changes. Remember that this config overrides appsettings.json so Heroku will try to use local non existing database